assembly-combined-package/bin/install.sh

```diff
@@ -18,6 +18,7 @@
 source ~/.bash_profile
 shellDir=`dirname $0`
 workDir=`cd ${shellDir}/..;pwd`
+common_conf=$LINKIS_HOME/conf/linkis.properties
```

Comment:
`common_conf=$LINKIS_HOME/conf/linkis.properties`
This line is unnecessary?

Reply:
yes, the following `sed`s  will modify this file